pkg/recommender.py

```diff
@@ -441,20 +443,57 @@ def _schedule_evt(self):
     Send the morning message at 10 am
     '''
     next_morning = ''
+    #Default message time
+    morn_hour = 10
+    morn_min = 0
+    ev_hour = 23
+    ev_min = 0
+
+    #get start time from deployment
+    try:
+      con = sqlite3.connect('C:/Users/Obesity_Project/Desktop/Patient-Caregiver Relationship/Patient-Caregiver-Relationship/DeploymentInformation.db')
+      cursorObj =  con.cursor()
+
+      table_name = 'RESIDENTS_DATA'
+      #select the latest deploymnet by ordering table by created date
+      cursorObj.execute("SELECT * FROM " + table_name + " ORDER BY created_date ASC LIMIT 1")
+      rows = cursorObj.fetchall()[0][11] # extract only start time
+      start_hour = int(rows[0:2])
+      start_minute = int(rows[3:5])
+    except Exception as e:
+      print(e)
+
+    con.close()
```

Comment:
seems you want to sth no matter it succeed or fail in a `try ... catch...`
you may consider `finally` here is the reference https://docs.python.org/3/tutorial/errors.html#defining-clean-up-actions

```
try:
  ...
except:
  ...
finally:
  # this will always run even u throw errors above

```